src/nbl/video/device_capabilities/device_limits.json

```diff
@@ -992,45 +990,45 @@
                 {
                     "type": "bool",
                     "name": "shaderUniformBufferArrayNonUniformIndexingNative",
-                    "value": false
+                    "value": true
                 },
                 {
                     "type": "bool",
                     "name": "shaderSampledImageArrayNonUniformIndexingNative",
-                    "value": false,
+                    "value": true,
                     "comment": ["promotion candidate"]
                 },
                 {
                     "type": "bool",
                     "name": "shaderStorageBufferArrayNonUniformIndexingNative",
-                    "value": false
+                    "value": true
                 },
                 {
                     "type": "bool",
                     "name": "shaderStorageImageArrayNonUniformIndexingNative",
-                    "value": false,
+                    "value": true,
                     "comment": ["promotion candidate"]
                 },
                 {
                     "type": "bool",
                     "name": "shaderInputAttachmentArrayNonUniformIndexingNative",
-                    "value": false,
+                    "value": true,
                     "comment": ["promotion candidate"]
                 },
                 {
                     "type": "bool",
                     "name": "robustBufferAccessUpdateAfterBind",
-                    "value": false
+                    "value": true
```

Comment:
if you want to require, then you need the `expose` tag

Also the `robustBufferAccessUpdateAfterBind`  will need special treatment, one should only require/check for it in CVulkanPhysicalDevice.cpp if the robust buffer access feature is enabled

Reply:
These are actually not supported on all profiles yet (even on my GPU). the profile merger script probably got confused because they exist on both `VkPhysicalDeviceDescriptorIndexingProperties` and `VkPhysicalDeviceVulkan12Properties`

Reply:
ouch